geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/pubsub/Subscription.java

```diff
@@ -18,6 +18,8 @@
 
 import java.util.List;
 
+import it.unimi.dsi.fastutil.objects.ObjectOpenCustomHashSet;
```

Comment:
I don't think this import is currently used and could be removed.  `spA` should handle that though.